OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -20,28 +22,37 @@ namespace OpenRA.Mods.Common.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class CaptureProperties : ScriptActorProperties
 	{
-		readonly CapturesInfo normalInfo;
+		readonly Captures[] capturesTraits;
```

Comment:
Style nit: we don't qualify trait variable names with "Traits".  Just call it `captures`.